app/controllers/reservation_frames_controller.rb

```diff
@@ -2,6 +2,7 @@ class ReservationFramesController < ApplicationController
   before_action :signed_in_user, only: [:new, :create]
 
   def index
+    @reservation_frames = current_user && ReservationFrame.where(planner_id: current_user.id)
```

Comment:
ここは関連付けで付けで撮ってこれるはずです！

```suggestion
    @reservation_frames = current_user.reservation_frames
```

Reply:
確かにそうですね 忘れていました ありがとうございます！